src/d_main.c

```diff
@@ -551,14 +551,10 @@ void D_DoAdvanceDemo(void)
   name = demostates[demosequence][gamemode].name;
   if (name && !strcasecmp(name, "TITLEPIC"))
   {
-    int i = W_CheckNumForName("TITLEPIC");
-    int j = W_CheckNumForName("DMENUPIC");
-
-    if (i < 0 || (j >= 0 && W_IsIWADLump(i)))
-      name = (j >= 0) ? "DMENUPIC" : "INTERPIC";
+    if (W_CheckNumForName("TITLEPIC") < 0)
+      name = "DMENUPIC";
```

Comment:
I didn't understand these checks.
Unity IWAD has both `TITLEPIC` and `DMENUPIC`.

Reply:
The Doom 2 IWAD that comes with the Doom 3 BFG Edition is missing the TITLEPIC lump and is supposed to show DMENUPIC instead.

Reply:
Yes, but why is there `W_IsIWADLump(i)`? Do my changes work too?

Reply:
If the IWAD provided both lumps, I wanted DMENUPIC to take precedence for some reason. 🤔

Reply:
Unity IWAD has both, but only TITLEPIC is wide.

Reply:
Ah, now that's a reason to prefer that instead!